### PR TITLE
Update masonry post, and add updated date to post headers

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Description
Uncomment if you want to provide a custom description.

- Updates to the recent masonry post
- Added an 'updated' property to the yaml data, which shows in a post header

## Related Issue(s)

- https://github.com/oddbird/oddleventy/issues/725

## Steps to test/reproduce

- Both can be seen on the masonry post, but I did add the metadata to a few other posts with clearly marked updates.

### CODE REVIEW

**Review by SondraE — APPROVED** [review]

File: content/blog/2024/grid-masonry-nuance.md [hunk 1]
```diff
@@ -571,3 +582,64 @@ So I'm rooting for the process!
...
+### Are we really calling this 'masonry'?
+
+At the end of the article,
+there is a suggestion that we re-think the name of this feature.
+The only real argument for 'masonry'
+is a popular tool with the same name --
+and a metaphor that's not particularly precise.
+
+I agree, but don't love their proposed `collapse` or `pack` values.
+Do you have better ideas?
```
Comment (stacyk) [inline content/blog/2024/grid-masonry-nuance.md:645]: Could we let them know where to send these ideas? Maybe add a link here to the preferred conversational channel.

**Review by stacyk — APPROVED** [review]
Left a few suggestions. The "updated" stuff looks/works well. I think that is helpful.

**Comment by mirisuzanne** [comment]
@stacyk how does that look?

**Review by stacyk — APPROVED** [review]
That works for me!

File: content/blog/2024/grid-masonry-nuance.md [hunk 2]
```diff
@@ -571,3 +582,66 @@ So I'm rooting for the process!
...
+They provide use-cases to show where the grid fallback is preferable,
+but also how a current masonry-like solution (similar to mine)
+could be progressively enhanced in much the same way as `subgrid`.
+That feeds into their argument that,
+while defaults are nice,
+most layouts will require more detail --
+and often that detail is where we get overlap with grids.
+So the simplest cases might be a line or two shorter
+with a non-grid approach,
+but realistic use-case are likely to duplicate the work.
```
Comment (jgerigmeyer) [inline content/blog/2024/grid-masonry-nuance.md:606]:
```suggestion
but realistic use-cases are likely to duplicate the work.
```

File: content/blog/2024/grid-masonry-nuance.md [hunk 3]
```diff
@@ -571,3 +582,66 @@ So I'm rooting for the process!
...
+
+Taking that even farther,
+they call into question how useful it is
+to have `auto` as a repeating value in masonry layouts.
+This caught me off guard --
+up above I say it sounds useful --
+but I think Apple is right two question that.
```
Comment (jgerigmeyer) [inline content/blog/2024/grid-masonry-nuance.md:613]:
```suggestion
but I think Apple is right to question that.
```

**Review by jgerigmeyer — APPROVED** [review]

**Pull request merged**